cinema/models.py

```diff
@@ -34,13 +39,21 @@
     def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
+    def movie_image_path(self, movie: "Movie", filename: str) -> pathlib.Path:
+        filename = (
+            f"{slugify(movie.title)}-{uuid.uuid4()}"
+            f"{pathlib.Path(filename).suffix}"
+        )
+        return pathlib.Path("upload/movies/") / pathlib.Path(filename)
+
 
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="movie_image_path")
```

Comment:
The `upload_to` parameter in the `ImageField` should reference a callable or a string path. Currently, it is set to the string 'movie_image_path', which suggests it might be intended to use the `movie_image_path` method defined in the `Actor` class. If this is the case, you should pass the callable directly without quotes, or ensure the path is correctly set as a string if it's a static path.